### PR TITLE
feat(ui-watt): add chips component

### PR DESCRIPTION
**Pull request by ManBearTM**
## Description

Simple implementation of chips, covering only the "choice chips variant". 

## References

- #487

### CODE REVIEW

File: libs/ui-watt/src/lib/components/chips/watt-chips.component.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * Simple implementation of Material "Choice chips".
+ * @see https://material.io/components/chips#choice-chips
+ */
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'watt-chips',
+  styleUrls: ['./watt-chips.component.scss'],
+  templateUrl: './watt-chips.component.html',
```
Comment (frontend-specialisten) [inline libs/ui-watt/src/lib/components/chips/watt-chips.component.ts:42]: Suggestion: Add `changeDetection: ChangeDetectionStrategy.OnPush`

Here are some articles you can take a look at: 
https://angular.io/api/core/ChangeDetectionStrategy
https://www.thinktecture.com/en/angular/whats-the-hype-onpush/
https://blog.angular-university.io/onpush-change-detection-how-it-works/

File: libs/ui-watt/src/lib/components/chips/watt-chips.stories.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+} as Meta<WattChipsComponent>;
+
+export const Overview: Story<WattChipsComponent> = (args) => ({
+  props: args,
+});
+
+Overview.args = {
+  selection: 'day',
+  options: [
+    { label: 'Dag', value: 'day' },
```
Comment (frontend-specialisten) [inline libs/ui-watt/src/lib/components/chips/watt-chips.stories.ts:39]: Suggestion: Change the labels to be english as we haven't used danish other places (at least what I know of). I think it's a good idea that `label` and `value` aren't the same. Maybe just change the value to be some kind of id instead?

**Review by frontend-specialisten — CHANGES_REQUESTED** [review]
Praise: Looks good! Only some minor comments :)

File: libs/ui-watt/src/lib/components/chips/watt-chips.component.ts (hunk 1 above)
Comment (ManBearTM) [reply libs/ui-watt/src/lib/components/chips/watt-chips.component.ts:44]: 0f6e24b7bdcc3c1e7b1b4743df4c68b8d0284371

File: libs/ui-watt/src/lib/components/chips/watt-chips.stories.ts (hunk 2 above)
Comment (ManBearTM) [reply libs/ui-watt/src/lib/components/chips/watt-chips.stories.ts]: 04ee9769471f6ba315058895cef8f91fb266a544

**Review by frontend-specialisten — APPROVED** [review]

**Pull request merged**